### DISCUSSION TITLE
Can mojo be compiled to a shared library?

### DISCUSSION THREAD

**Discussion by azkadev** [modular/modular#107] [Q&A] [answered]
Most open source ai projects are built in python,

I want to add ai to my app.

So, can Mojo be compiled later into shared libraries?

sorry for the question and my english is bad

**Comment by lattner** [comment, marked as answer]
Yes, it can be compiled as a shared library, no problem.  We're not prioritizing this right now, but we'll enable this at some point

**Comment by azkadev** [comment]
wow amazing, i wish mojo could become language 1 and could replace python quickly,
